spec/controllers/heroku/resources_controller_spec.rb

```diff
@@ -52,11 +55,31 @@
 
       expect(sandwich.oauth_grant_code).to eq(code)
     end
+
+    it 'enqueues the ExchangeGrantTokenJob' do
+      http_login(ENV['SLUG'], ENV['PASSWORD'])
+      stub_grant_code_exchanger
+      heroku_uuid = '123-ABC-456-DEF'
+      code = 'supersecretcode'
+
+      post :create, params: {
```

Comment:
We don't need any special headers to work with `id.heroku.com`?

Reply:
I think the POST is just form params for the OAuth spec.

We'd need headers and a JSON POST body if we created the OAuth token in `api.heroku.com`

Reply:
I test this via postman and it worked with just the params